### DISCUSSION TITLE
How do can I use react-pdf with @react-pdf/renderer for live preview of pdf file?

### DISCUSSION THREAD

**Discussion by joseDaKing** [wojtekmaj/react-pdf#887] [Q&A] [answered]
How can I use react-pdf with @react-pdf/renderer for a live preview of the pdf file?

**Comment by wojtekmaj** [comment, marked as answer]
Interesting question! Actually, this is exactly what's being done in [`@react-pdf/renderer`'s Playground / REPL](https://react-pdf.org/repl). 

Here's the source code:
https://github.com/diegomura/react-pdf-site/tree/master/src/components/Repl

Does that help?

**Reply by joseDaKing** [reply]
I sincerely thank you for the help because you cannot imagine how much time I have wasted on this

**Comment by kksingla** [comment]
@wojtekmaj thanks for the great library.

In a live update, can we remove flicker?

similar to https://resume.io/app
